wrapper.go

```diff
@@ -30,32 +30,32 @@ const (
 )
 
 // KeyType defines types of cryptographic keys.
-type KeyType string
+type KeyType int
 
 const (
-	RSA2048 KeyType = "rsa-2048"
-	RSA3072 KeyType = "rsa-3072"
-	RSA4096 KeyType = "rsa-4096"
+	RSA2048 KeyType = iota
```

Comment:
You might want to use `1 + iota` to avoid the zero-value of KeyType (and the other types below) defaulting to the first value on the const block.

Reply:
Good idea. Done in 749f8da.